harvester/build.gradle

```diff
@@ -1,38 +1,29 @@
 buildscript {
-
     dependencies {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:${version_springBoot}"
-        classpath 'com.sourcemuse.gradle.plugin:gradle-mongo-plugin:0.11.0'
+        classpath "com.sourcemuse.gradle.plugin:gradle-mongo-plugin:${version_mongo_plugin}"
```

Comment:
Let's decide whether we use camelCase or underscores for properties and use that same style everywhere. We should also document these style guides on a wiki page.